packages/compass-query-bar/src/stores/ai-query-reducer.ts

```diff
@@ -39,6 +41,8 @@ export const enum AIQueryActionTypes {
   ShowInput = 'compass-query-bar/ai-query/ShowInput',
   HideInput = 'compass-query-bar/ai-query/HideInput',
   ChangeAIPromptText = 'compass-query-bar/ai-query/ChangeAIPromptText',
+  ShowOptIn = 'compass-query-bar/ai-query/ShowOptIn',
+  HideOptIn = 'compass-query-bar/ai-query/HideOptIn',
```

Comment:
```suggestion
```

Reply:
Ah ok  :)

Reply:
Yeah, moved them to sign in reducer, and forgot to clean up here, good catch!